src/sentry/sentry_apps/api/parsers/sentry_app.py

```diff
@@ -95,7 +146,7 @@ class SentryAppParser(Serializer):
 
     def __init__(self, *args, **kwargs):
         self.active_staff = kwargs.pop("active_staff", False)
-        self.access = kwargs.pop("access")
+        self.access = kwargs.pop("access", None)
```

Comment:
This change is the sketchy one, so I added an explicit assertion to the scope validation method which will raise in a similar manner to this one, but with a more explicit assertion.

Reply:
why is it a sketch change ? Wouldn't the behavior be the same ?

Reply:
Yeah it would still blow up with a key error further down the validation pipeline, so not super sketchy. I just don't like the idea of not requiring an `access` property if it's required for the serializer to validate permissions correctly.